index.bs

```diff
@@ -1959,7 +1965,13 @@ The <dfn method for="XRWebGLLayer">getViewport(|view|)</dfn> method, when invoke
   1. If |frame|'s {{XRFrame/session}} is not equal to |layer|'s [=XRWebGLLayer/session=], throw an {{InvalidStateError}} and abort these steps.
   1. If |frame|'s [=XRFrame/active=] boolean is <code>false</code>, throw an {{InvalidStateError}} and abort these steps.
   1. Let |glViewport| be the [=WebGL viewport=] from the [=list of viewports=] associated with |view|.
-  1. Let |viewport| be a [=new=] {{XRViewport}} in the [=relevant realm=] of |layer|'s [=XRWebGLLayer/session=].
+  1. Initialize |viewport| as follows:
+    <dl class="switch">
+      <dt>If a |view| with the same underlying [=view=] has been passed to a getViewport() call previously, and |glViewport| has not changed since then, the user agent MAY:</dt>
```

Comment:
```suggestion
      <dt>If a |view| with the same underlying [=view=] has been passed to this |layer|'s getViewport() call previously, the user agent MAY:</dt>
```

Is "with the same underlying [=view=]" needed?

Reply:
Probably, since `XRView`s aren't a good candidate for caching. (The `transform` and `projectionMatrix` make them closer to a POD structure like `XRPose` than an opaque key, though I made that same mistake in the previous PR)

Reply:
@cabanier yes, you have a different XRView object each frame